src/GeolocatorService.Reactive/GeolocatorServiceExtensions.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Linq;
+using System.Reactive.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Windows.Devices.Geolocation;
+
+namespace GeolocatorService
+{
+	public static class GeolocatorServiceExtensions
```

Comment:
i think all the methods are missing the "this" keyword.

should be `public static IObservable<Geocoordinate> ObserveLocation(this IGeolocatorService service)`